spec/features/cache_headers_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe "Verifying Cache-Control in headers", :type => :request do
+  context "Requests should get correct cache in response headers" do
+    it 'should have headers that prevent caching profile' do
+      get '/profile'
+      expect(response.headers["Cache-Control"]).to include("no-cache")
+    end
+
+    it 'should have headers with public caching assets' do
+      get '/assets/bg-login.jpg'
+      expect(response.headers["Cache-Control"]).to include("public")
+    end
+
+    it 'should have headers with private caching in login' do
+      get '/login'
+      expect(response.headers["Cache-Control"]).to include("max-age=0", "private")
```

Comment:
any idea why `/login` and `/profile` have different cache-control headers?

Reply:
I think it might be because we're not mocking a user...